### PR TITLE
Update redirect guidance in msal-react

### PR DESCRIPTION
**Pull request by tnorling**
Expands msal-react FAQ entry regarding the handling of redirects

### CODE REVIEW

**Review by jo-arroyo — APPROVED** [review]

File: lib/msal-react/FAQ.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ If you need direct access to the response object or error returned by a redirect
 
 1. Use the [event API](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-react/docs/events.md) to register a callback that will be invoked with the response when you return to your app. Make sure this is registered in a code path that is run __after__ the redirect as any callbacks registered __before__ the redirect will be lost.
 1. Use the [useMsalAuthentication hook](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-react/docs/hooks.md#usemsalauthentication-hook) to login. This hook will return the result or error when you are returned to your app.
-1. Call `handleRedirectPromise` which will resolve with the result or reject with an error if the page was loaded as a result of a redirect
+1. Call `handleRedirectPromise` which will resolve with the result or reject with an error if the page was loaded as a result of a redirect.
```
Comment (sameerag) [inline lib/msal-react/FAQ.md:75]: What happens if the page isn't reloaded? Just curious if we need to document this or if it ever happens.

**Review by sameerag — APPROVED** [review]
Approving. Minor comment.

File: lib/msal-react/FAQ.md (hunk 1 above)
Comment (tnorling) [reply lib/msal-react/FAQ.md:75]: What do you mean by "reloaded"? If the page is loaded after a redirect you'll get the result or error, if not (i.e. load up the page for the first time) you'll get null

Comment (sameerag) [reply lib/msal-react/FAQ.md:75]: Yeah, I meant if it never redirects - `load up the page first time` use case. Thanks.

**Pull request merged**